src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.workspace.common.exception.ValidationException;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ControllerValidationUtils {
+
+  public static void ValidatePaginationParams(int offset, int limit) {
+    List<String> errors = new ArrayList<>();
+    if (offset < 0) {
+      errors.add("offset must be greater than or equal to 0.");
+    }
+    if (limit < 1) {
+      errors.add("limit must be greater than or equal to 1.");
+    }
```

Comment:
what's your opinion: should we add a sanity-check upper bound for page size?  Else, users could request limit=999999999, which could be bad for the system.

Conversely, should there be a way for a user to specify they truly want all results?

If we start building any extensive validation, let's consider pulling in a library like apache commons validator, or something.

Reply:
I'm a little wary of adding more specific validation logic to shared utils like this, but I'm also not a fan of leaving limit unbounded. 
I'd lean towards leaving it unbounded for now and changing it if we expect users to call this directly, but I'm willing to change that.